### PR TITLE
Create resident-architecture.md

### PR DESCRIPTION
**Pull request by anorrish**

### CODE REVIEW

File: content/terms/resident-architecture.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- Resident Architect - Standard
+  - 260 total annual hours (approximately 5 hours per week)
+  - Primarily focused on custom onboarding and enablement work.
+- Resident Architect - Advanced
+  - 520 total annual hours (approximately 10 hours per week)
+  - Additional time beneficial for more hands on configuration work and more technical requirements from the service.
+- Resident Architect - Premium
+  - 1,040 total annual hours (approximately 20 hours per week)
+  - Good for highly technical, intensive projects often requiring hands-on custom development.
+
```
Comment (alexfogg) [inline content/terms/resident-architecture.md:55]: I'd maybe add a sentence like "While we recommend distributing the hours evenly throughout the year, the total hours are allocated as an annual pool. This allows for flexibility in usage, enabling you to utilize the hours as needed to best align with your project timelines and priorities."

File: content/terms/resident-architecture.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+### Consumption Rules
+The parties agree that the following rules shall apply to the consumption of hours set forth above:
+  - (a) Availability: Unless otherwise agreed by the parties in writing, Professional Services will be performed Monday through Friday, between the hours of 8:00 a.m. – 7:00 p.m. in the Customer’s local time zone. Work outside of the hours above, work on weekends or on local holidays must be pre-approved in writing by Sourcegraph.
+  - (b) Location: Professional Services will be performed at a remote location.
+  - (c) Multiple Personnel: Sourcegrah may, at its sole discretion, choose to engage different consulting personnel for different portions of the Professional Services.
```
[the remark this answers is not in the document]
Comment (anorrish) [reply content/terms/resident-architecture.md]: Fixed

File: content/terms/resident-architecture.md (hunk 1 above)
Comment (anorrish) [reply content/terms/resident-architecture.md:55]: Does this comment from early in the terms cover this:

"Note: hours may be able to flex monthly, subject to planning and based on project needs but should not exceed the annual allocation. Time spent towards the service will be tracked discretely and reported to Finance monthly."

Comment (alexfogg) [reply content/terms/resident-architecture.md:55]: Yes!

**Review by alexfogg — APPROVED** [review]

**Pull request merged**